tutorials/commands/index.rst

```diff
@@ -6,8 +6,9 @@ Command Tutorials
 Welcome to the Command Tutorials! These tutorials should help you to map specific Flux commands
 with your use case, and then see detailed usage. 
 
- - ``flux proxy`` (:ref:`ssh-across-clusters`): "Send commands to a flux instance across clusters using ssh"
  - ``flux-mini-submit`` (:ref:`flux-mini-submit`): "Submit a job in a Flux instance"
+ - ``flux proxy`` (:ref:`flux-proxy-command`): "flux proxy basics"
```

Comment:
Can we here give the reader a better sense of why they would want to use flux proxy, and how it differs from the "ssh across clusters" tutorial?